tests/monotouch-test/Foundation/FileManagerTest.cs

```diff
@@ -120,6 +129,9 @@ public void DefaultManager ()
 		}
 
 		[Test]
+#if MONOMAC
```

Comment:
Should we #if the [Test] out instead since this will never be fixed? That way it doesn't show up as ignored?